zaplings/views.py

```diff
@@ -778,30 +794,42 @@ def process_rsvp(request):
-    name = request.POST['user-firstname'] or 'there'
-    response = requests.post(
-        "https://api.mailgun.net/v2/mg.zaplings.com/messages",
-        auth=("api", "key-9c1bcab96768c19c80966e42c7882cd0"),
-        data={
-            "from": "Zaplings Team <postmaster@mg.zaplings.com>",
-            "to": request.POST['user-email'],
-            "subject": "See you at Creators' Night!",
-            "h:Reply-To": "danny@zaplings.com",
-            "html": email_body % name    
-        }
-    )
+    email_body = '<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html> <head> <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/> <title>Zaplings - Creators&#39; Night confirmation</title> <style>@media only screen and (min-device-width: 641px){.content{width: 640px !important;}}</style></head> <body><!--[if (gte mso 9)|(IE)]> <table width="540" align="center" cellpadding="0" cellspacing="0" border="0"> <tr> <td><![endif]--> <table class="content" align="center" cellpadding="0" cellspacing="0" border="0" style="width: 100%%; max-width: 640px; font-family: Arial;"> <tr bgcolor="#05325b" style="text-align: center;"> <td valign="bottom"> <a href="http://www.zaplings.com"><img src="/static/images/logo-email.png" alt="Zaplings" width="111px" height="50px"/></a> </td></tr><tr align="left" style="color: #05325b;"> <td style="padding: 48px;"> Hi %s,<br><br>We&#39;re excited that you will be joining us for the next Creators&#39; Night. Here&#39;s a reminder of what&#39;s in store:<br><br><strong>Featured creators</strong> <ul> <li>Music by <a href="https://shanecooley.bandcamp.com/" target="_blank">Shane Cooley &amp; The Lucky Kings</a></li><li>Art by <a href="http://www.rexhamiltonart.com" target="_blank">Rex Hamilton</a></li><li>Startup founder of <a href="http://www.peepsqueeze.com" target="_blank">Peepsqueeze</a></li><li>Photography by <a href="http://www.mynameiskat.com" target="_blank">Kat Goins</a></li></ul> <strong>Opportunities for you to create</strong> <ul> <li>Express yourself with 5-minute mic time</li><li>Brainstorm at the idea table</li><li>Make art at the bARTer table</li><li>Grow friendships with connection cards</li><li>Sell or bid in the art auction</li></ul> <p>Thank you for your help in growing this community of creators. Forward this email to friends who might be interested. Remind them to Rsvp at <a href="http://www.zaplings.com/creatorsnight" target="_blank">zaplings.com/creatorsnight</a> and arrive by 6:45pm to ensure entry.</p><p>Create on!<br>The Zaplings Team</p></td></tr></table><!--[if (gte mso 9)|(IE)]> </td></tr></table><![endif]--> </body></html>'
+    
+    zaplings_emails = [
+        'nicko.shestopalov@gmail.com',
+        'danny@zaplings.com',
+        'dannypernik@gmail.com',
+        'drpernik@gmail.com',
+        'justin.cole1981@gmail.com',
+    ]
+
+    user_email = request.POST['user-email']
+    if user_email in zaplings_emails:
```

Comment:
This is the change we talked about in hangout so that at this time we send confirmation emails to us only (for testing purposes).